docs/source/cleanlab/datalab/guide/issue_type_description.rst

```diff
@@ -141,6 +141,18 @@ To find the underperforming group, Cleanlab clusters the data using the provided
 The underperforming group quality score is equal to `q/r` for examples belonging to the underperforming group, and is equal to 1 for all other examples.
 Advanced users:  If you have pre-computed cluster assignments for each example in the dataset, you can pass them explicitly to :py:meth:`Datalab.find_issues <cleanlab.datalab.datalab.Datalab.find_issues>` using the `cluster_ids` key in the `issue_types` dict argument.  This is useful for tabular datasets where you want to group/slice the data based on a categorical column. An integer encoding of the categorical column can be passed as cluster assignments for finding the underperforming group, based on the data slices you define.
 
+Null Issue
+-----------
+
+Examples flagged with null issue have null/missing values across all feature columns.
+
+Null issues are detected based on provided `features`.  If you do not provide the `features`, this type of issue will not be considered.
+
+The null issue quality score for each example is the proportion of features values that are not null. The overall null issue quality score
+is the mean of the individual quality scores.
+
+Presence of null examples in the dataset can lead to errors when training ML models. It can also
+result in the model learning incorrect patterns due to the null values.
```

Comment:
```suggestion
Examples identified with the null issue correspond to rows that have null/missing values across all feature columns (i.e. the entire row is missing values).

Null issues are detected based on provided `features`.  If you do not provide `features`, this type of issue will not be considered.

Each example's null issue quality score equals the proportion of features values in this row that are not null/missing. The overall dataset null issue quality score
equals the average of the individual examples' quality scores.

Presence of null examples in the dataset can lead to errors when training ML models. It can also
result in the model learning incorrect patterns due to the null values.
```